### ISSUE TITLE
In an IF expression lines move to a wrong place

### ISSUE THREAD

**Issue by d-biehl** [MarketSquare/robotframework-tidy#226]
Hi,

Given this robotframework code:

```robotframework
*** Keywords ***
First
    IF    ${i}==1
    Log    "one"
    END

*** Test Cases ***
First
    IF    ${i}==1
    Log    "one"
    END
```

running robotidy with default (means no arguments, config file or something), gives this:

```robotframework
*** Test Cases ***
First
    IF    ${i}==1
    END
    Log    "one"  # << this is wrong

*** Keywords ***
First
    IF    ${i}==1
    END
    Log    "one"  # << this is wrong
```

Tested with:
robotidy 1.6.1, 1.6.0
python 3.8, 3.10
windows, linux

robotidy 1.5.1 works correctly.

**Comment by bhirsz** [comment]
Thats..interesting for sure. In last versions one of the additions was transformer that is adding missing END - can you run robotidy with disabled AddMissingEnd transformer to verify if its the cause? 
```
robotidy - c AddMissingEnd:enabled=True
```

**Comment by d-biehl** [comment]
with ```robotidy -c AddMissingEnd:enabled=False``` it works correctly.

But there is no missing END?

**Comment by bhirsz** [comment]
Thanks for confirming. Yeah but I'm not sure if robot Framework recognize keywords inside such if statement as part of if - if there is no indent. If it doesnt recognize them then it would make sense why robotidy add end marker. I will investigate it because it would be better if it would be fixed (properly intended) instead

**Comment by d-biehl** [comment]
Ok, I'll just make it a real running robot file:

```robotframework
*** Keywords ***
First Keyword
    ${i}    Set Variable    1
    IF    ${i}==1
    Log    "one"
    END

*** Test Cases ***
First
    ${i}    Set Variable    1
    IF    ${i}==1
    Log    "one"
    END
    First Keyword
```

runs fine with robot, but formats wrong with robotidy.

![project1-1636405684731](https://user-images.githubusercontent.com/7069968/140818594-c77f59ad-5f52-433b-ba05-4d92f5295cab.gif)

for robotframework indentation does'nt matter for regocnizing IF expressions. The parser finds and IF keyword and then looks for ELSE(IF)/END.

**Comment by bhirsz** [comment]
Fixed in linked PR - should be released as 1.6.2 as soon as this and other PR will be merged.
The cause of the bug was missing check if the IF does need 'fixing' missing END at all 😄 (don't fix if it doesn't need fixing ;) )
